jenkins/hipchat.sls

```diff
@@ -0,0 +1,10 @@
+/srv/jenkins/jenkins.plugins.hipchat.HipChatNotifier.xml:
+  file.managed:
+    - user: jenkins
+    - group: jenkins
+    - mode: 644
+    - source: salt://jenkins/templates/jenkins.plugins.hipchat.HipChatNotifier.xml
+    - template: jinja
+    - require:
+      - user: jenkins
+
```

Comment:
This needs to be optional as not every user of this formula will have or want hipchat.

Follow the pattern of using map.jinja like this: https://github.com/ministryofjustice/jenkins-formula/blob/3c0cd231928c56bc371f38299eaf9a8acd890298/jenkins/user.sls#L9-L11 and https://github.com/ministryofjustice/jenkins-formula/blob/cbe880ec0f0fa279f1e927287cd411e8a9a35761/jenkins/map.jinja#L16